### DISCUSSION TITLE
set default converter

### DISCUSSION THREAD

**Discussion by lazyclaw-labs** [linq2db/linq2db#3480] [General]
I would like to have a default converter when the system type convert does not work.
i.e. I have a varchar in db, and I wang to define my c# class property as any Type, i.e., Address, or Contact ... etc.
Then Linq2DB will throw exception said it can't convert from string to my type: Address, Contact, or whatever.

The easy solution is to add a setconverter for each of the type I want to deal with.
But it will be really convenient if I can simply say: Serialize to Json, Deserialize From json. Thus I don't have to set converter for all the types.
So if linqtodb can give me a place to specify defaultconverter when the system can't decide by itself, it will be really great. i.e., let me intercept when you can't convert, don't just throw exception.

**Comment by MaceWindu** [comment]
There is no such converter, but you can try to play with object-to-object converter
